components/layout/blog-view.hbs

```diff
@@ -0,0 +1,226 @@
+<div class="section--bg-white blog-feed">
+    <a href="#" class="content-block--clickable blog-content">
+        <div class="grid-col-content__two featured">
+            <div class="content-block">
+                <img class="content-block__image" src="https://www.fillmurray.com/g/545/747"
+                    role="presentation" alt="" />
+            </div><!-- .content-block -->
+
+            <div class="content-block">
+                <h1 class="content-block__title anchor-primary">
+                    4 ways to keep your talent during the Turnover Tsunami.
+                </h1>
+
+                <div class="blog-author-date">
```

Comment:
This also feels a little out of convention. Here and below on 23, you're declaring these as two very specific blocks, even though they're really elements of the blog block. I'm kinda feeling like maybe what we need is a `content-block__blog-post` or something like that at the top. Then everything else flows as `content-block__blog-author" etc. Or we just create a new block `blog-post` that extends entirely off `content-block`.